docs/reference/configuration-reference/alerting-settings.md

```diff
@@ -134,6 +134,22 @@ $$$action-config-email-domain-allowlist$$$
 
     Data type: `string`
 
+`xpack.actions.email.services.ses.host` ![logo cloud](https://doc-icons.s3.us-east-2.amazonaws.com/logo_cloud.svg "Supported on {{ech}}")
+:    The SMTP endpoint for an Amazon Simple Email Service (SES) service provider that can be used by email connectors.
+
+    ::::{warning}
+    This setting alone is insufficient to override system defaults for the SES SMTP endpoint. You must also configure the `xpack.actions.email.services.ses.port` setting
```

Comment:
Hey @nastasha-solomon, could you review it again, please? I did some changes + this warning section is new

Reply:
Just a couple minor edits - thanks!

```suggestion
    This setting alone is insufficient for overriding system defaults for the SES SMTP endpoint. You must also configure the `xpack.actions.email.services.ses.port` setting.
```

Reply:
done in 269bc6b572aaec775d9fb008892c76fb63b8c38e